docs/customization.rst

```diff
@@ -298,6 +298,11 @@ Configuration Options
             Note that if the ``toc.integrate`` feature is enabled, the left
             sidebar is instead scrolled for the local table-of-contents as
             described above.
+
+          .. warning::
+              When enabling the ``toc.follow`` feature,
```

Comment:
Oops. Looks like my suggestion's indent was off somehow.

```suggestion
             When enabling the ``toc.follow`` feature,
```
Currently, it renders as a block quote.